FrontEndReact/src/View/Admin/View/ViewAssessmentTask/ViewAssessmentTasks.js

```diff
@@ -133,13 +138,34 @@ class ViewAssessmenTasks extends Component {
                             <p
                                 className='mt-3'
                                 variant="contained"
+                                align="center"
                             >
                                 {suggestions ? (suggestions ? "Yes" : "No") : "No"}
                             </p>
                         )
                     }
                 }
             },
+
+            // currently hardcoded to default to no as the column is not connect to the database
```

Comment:
Update the comment to say: "Currently we hardcoded the "Team Assessment?" column to "No" because the database has not been updated to store actual data."